core-bundle/contao/classes/Versions.php

```diff
@@ -721,6 +724,31 @@ public static function addToTemplate(BackendTemplate $objTemplate)
 				$arrRow['editUrl'] = $request->getBasePath() . '/' . preg_replace(array('/&(amp;)?popup=1/', '/&(amp;)?rt=[^&]+/'), array('', '&amp;rt=' . htmlspecialchars(System::getContainer()->get('contao.csrf.token_manager')->getDefaultTokenValue(), ENT_QUOTES | ENT_SUBSTITUTE | ENT_HTML5)), StringUtil::ampersand(ltrim($arrRow['editUrl'], '/')));
 			}
 
+
+			$arrRow['operations'] = System::getContainer()->get('contao.data_container.operations_builder')->initialize('tl_version');
+
+			if ($arrRow['deleted'] ?? null) {
```

Comment:
This can not really work as we mark them as deleted later

Reply:
Before:
<img width="404" height="308" alt="image" src="https://github.com/user-attachments/assets/b72e454f-eb5e-47fa-b7ec-f7c56d6a656d" />

This PR:
<img width="425" height="354" alt="image" src="https://github.com/user-attachments/assets/e797e4de-5e9a-4ae1-afb7-ba71b267c681" />